cmds/fixmynetboot/dmesg.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+// shamelessly copied from u-root/cmds/dmesg
+const (
+	_SYSLOG_ACTION_READ_ALL = 3
```

Comment:
should we pay attention to the sticker ci here?

Reply:
not really, this is a non-Go constant

Reply:
https://elixir.bootlin.com/linux/v3.2/source/include/linux/syslog.h#L31